app/views/bookings/_form.html.haml

```diff
@@ -0,0 +1,16 @@
+= form_for @booking do |f|
+  - if @booking.errors.any?
+    #error_explanation
+      %h2= "#{pluralize(@booking.errors.count, "error")} prohibited this booking from being saved:"
```

Comment:
My objection is that this will evaluate to

error prohibited this booking from being saved:

or 

errors prohibited this booking from being saved:

Not 

1 error prohibited this booking from being saved:

or 

2 errors prohibited this booking from being saved:

I would also like to see this extracted to a helper tbh